torch_xla/csrc/tensor.cpp

```diff
@@ -1840,6 +1840,11 @@ c10::SymIntNode XLASymIntNodeImpl::mul(const c10::SymIntNode& other) {
   return c10::make_intrusive<XLASymIntNodeImpl>(nmul);
 }
 
+c10::SymIntNode XLASymIntNodeImpl::wrap(int64_t num) {
```

Comment:
I wonder why this is a member function of the class `XLASymIntNodeImpl` as opposed to a standalone util function. This function doesn't seem to use any information from the class itself.